.github/workflows/test.yml

```diff
@@ -216,9 +216,10 @@ jobs:
         uses: "./.github/actions/count_deps"
 
       - name: Store Verdaccio logfile artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
This change was made on the `develop` branch previously, is now porting over to `release-2.9.x` branch, see https://github.com/SalesforceCommerceCloud/pwa-kit/commit/990a132ee12e379b18bdbf96a786fd95be1ef913